subjects/gaming/stealth-boom/audit/README.md

```diff
@@ -1,20 +1,10 @@
 > Due to file size reason, the solution might be uploaded on GitHub instead of Gitea!
 
-#### Functional
+#### Mandatory
 
-###### Does the map contain places for the player to hide from enemies?
+##### Main Menu
 
-###### Does the map contain buildings, pickable ammunition and weapons placed around the map?
-
-###### Does the player have all the minimal animation required (walking, running, melee attacking, aiming, reloading, shooting, crouching, crouch walking, crouch aiming, crouch reloading, crouch shooting)?
-
-###### Is there a sound for the player shooting?
-
-###### Are there bullets impacts present when shooting at a wall?
-
-###### When the player is hit, are there any blood particles?
-
-###### Is a main menu with the five options displayed on the screen?
+###### Is the main menu with the five options displayed on the screen?
 
 ###### Can the general sound of the game be managed directly on the settings menu?
 
```

Comment:
`When changing the resolution, does a confirmation pop-up appear in the center of the screen asking if the player wants to keep the new graphics settings?`

`If the player presses 'No' on the graphics confirmation pop-up, or if the pop-up is not confirmed within 10 seconds, do the settings revert to the previous ones?`

Reply:
changed